engine/src/main/battlecode/world/GameMapIO.java

```diff
@@ -338,7 +338,9 @@ private static void initInitialBodiesFromSchemaBodyTable(SpawnedBodyTable bodyTa
                 int bodyY = locs.ys(i);
                 Team bodyTeam = TeamMapping.team(bodyTable.teamIDs(i));
                 if (bodyType != null)
-                    initialBodies.add(new RobotInfo(bodyID, bodyTeam, bodyType, new MapLocation(bodyX, bodyY)));
+                    initialBodies.add(new RobotInfo(bodyID, bodyTeam, bodyType, 0,
```

Comment:
does this give the hq a 10 turn delay?

Reply:
nope! when spawning robots only ID, team, type and location is used. I added a comment to clarify that.

thanks tho!